packages/file-type-icons/src/FileIconType.ts

```diff
@@ -11,7 +11,8 @@ export enum FileIconType {
   genericFile = 3,
   listItem = 4,
   sharedFolder = 5,
-  multiple = 6
+  multiple = 6,
+  stream=7
```

Comment:
Same formatting change as in the other PR
```suggestion
  stream = 7
```

Reply:
Done!